### PR TITLE
[HTM-38] Add database connectivity for Tailormap API

### PR DESCRIPTION
**Pull request by mprins**
Uses the original `org.tailormap:viewer-config-persistence` module for now and only for a preconfigured PostgreSQL config database.

resolves HTM-38

### CODE REVIEW

**Comment by mprins** [comment]
> Code scanning results / dependency-check 

failure is expected due to how that tool works and should be fixed after upgrading OWASP check plugin - which is currently in release stage

**Comment by mprins** [comment]
Not sure what is going on with codecov

File: src/main/resources/tailormap-api.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ info:
```
[the remark this answers is not in the document]
Comment (geertplaisier) [reply src/main/resources/tailormap-api.yaml:23]: Ok, looks fine. On URL's, I was thinking, is it useful to add a version in there? so /api/v1/  or something? If, in the future, we would like to implement newer versions of the API they can co-exists while developing?

[the remark this answers is not in the document]
Comment (mprins) [reply src/main/resources/tailormap-api.yaml:23]: I did initially specify that, but I think that would be a very small change in the code and a bigger change in deployment... I will created a backlog item so we can revisit: https://b3partners.atlassian.net/browse/HTM-54

**Review by matthijsln — APPROVED** [review]

**Pull request merged**